test/input/closeness_centrality_unweighted.gt

```diff
@@ -0,0 +1,58 @@
+element Vertex end
+element Edge end
+
+const edges : edgeset{Edge}(Vertex, Vertex) = load ("../../../../baghdadi/data3/socLive/socLive_gapbs.sg");
```

Comment:
Again, double check the path you set before committing changes. Make sure all the programs pass all the tests before you commit.